deploy-manage/deploy/cloud-on-k8s/managing-deployments-using-helm-chart.md

```diff
@@ -88,23 +88,141 @@ Enterprise Search is not available in {{stack}} versions 9.0 and later. For an e
+
+### Upgrade to specific {{stack}} version
+
+If for example you want to upgrade the {{stack}} components to a later version that is not the default for the Helm chart, you can explicitly set the component versions using Helm values or `--set` options.
+
+The following examples show both ways to upgrade the release to the latest available version of the Helm chart and all {{stack}} components to version {{version.stack}}.
+
+#### Using the `--set` option
+
+Use `--set` options to override the component versions directly from the command line:
+
+```sh
+helm repo update <1>
+helm upgrade es-kb-quickstart elastic/eck-stack -n elastic-stack \
+  --set eck-elasticsearch.version={{version.stack}} \ <2>
+  --set eck-kibana.version={{version.stack}}
+```
+1. Refresh the local chart cache.
+2. Specify versions for all the components you deploy. Components without an explicitly defined version continue to use the default versions provided by the chart.
+
+#### Using a values file
+
+If you already use a values file for this release, update it to include the following settings. Otherwise, create a new values file (for example, `custom-values.yaml`) with the following content:
+
+```yaml subs=true
+eck-elasticsearch: <1>
+  version: {{version.stack}}
+eck-kibana:
+  version: {{version.stack}}
+```
+1. Specify versions for all the components you deploy. Components without an explicitly defined version continue to use the default versions provided by the chart.
+
+Then upgrade the release using the values file:
+
+```sh
+helm repo update <1>
+helm upgrade es-kb-quickstart elastic/eck-stack -n elastic-stack -f custom-values.yaml
+```
+1. Refresh the local chart cache.
+
+### Apply configuration changes
+
+To apply configuration changes to an existing release, run `helm upgrade` with the complete configuration you want the release to use. This includes both the current configuration and any new changes.
+
+For example, if you installed the [quickstart release](#k8s-install-elasticsearch-kibana-helm) and want to scale the {{es}} cluster to three nodes and expose the {{kib}} service using a LoadBalancer, do the following:
+
+1. Create a values file with the desired configuration, and save it as `custom-values.yaml`:
+
+```yaml
+eck-elasticsearch:
+  nodeSets:
+  - name: default
+    count: 3
+
+eck-kibana:
+  http:
+    service:
+      spec:
+        # This deploys a load balancer in a cloud service provider, where supported.
+        type: LoadBalancer
+```
+
+2. Apply the configuration using `helm upgrade`:
+
+```sh
+helm upgrade es-kb-quickstart elastic/eck-stack \
+  -n elastic-stack \
+  -f custom-values.yaml
+```
+
+::::{warning}
+The previous example also upgrades the {{stack}} components if a newer Helm chart version is available. To avoid this, [identify the chart version](#show-versions) currently used by your release and include the `--version` option when running `helm upgrade`.
```

Comment:
because it's all grouped under one heading with only one code example, we can just say `this`

```suggestion
This example also upgrades the {{stack}} components if a newer Helm chart version is available. To avoid this, [identify the chart version](#show-versions) currently used by your release and include the `--version` option when running `helm upgrade`.
```